citest/reporting/html_index_renderer.py

```diff
@@ -119,7 +119,7 @@ def __handle_generic(self, entry):
     if entry.get('control') == 'END':
         # pylint: disable=bad-indentation
         self.__depth -= 1
-        if self.__depth == 0 and self.__in_test:
+        if self.__depth == 0:
```

Comment:
I'm not sure I understand why the `and self.__in_test` was removed.  The first commit commented it, and the second removed it entirely (whereas I was expecting the second commit to just uncomment)

Reply:
The clause used to be there because I only wanted to consider test failures.

When I added the CLI, it is pushing journals that arent running tests because the whole point is that it is reporting that it could not initiate the test. I commented out that clause to see if I needed it before I yanked it, but forgot to yank it.

Note that when I added the earlier change to catch issues in the setup/teardown which is also outside the tests, that was pretty much having the same effect of annotating top-level entities that arent in tests (but was outside contexts as well).

The net effect here is that rending journals as HTML isnt really about tests. It's really just a structured log file, which was the intent all along. But it isnt practical to think of it completely generalized because it isnt efficient and has limited expressiveness. It's still specialized for citest needs.

Reply:
Thanks for the explanation!  This makes perfect sense now.